### ISSUE TITLE
Overfitting

### ISSUE THREAD

**Issue by karimmahalian** [songyouwei/ABSA-PyTorch#203]
Hi,

I tried this code for another dataset but there is a big difference between training ACC and testing ACC (about 10%) I tried to modify the dropout value to reduce this overfitting but it did not work for me.

Any ideas plz?

thank u
